framework/resources/Functional/filter/pushdown/general/data.json

```diff
@@ -0,0 +1,27 @@
+{
+  "testId": "filter_pushdown_data",
```

Comment:
Can you please ensure the Test IDs across the PR are unique. I found this and the below file to share the IDs.

Reply:
Thanks!